### DISCUSSION TITLE
Can I help with English

### DISCUSSION THREAD

**Discussion by connect2400** [SoftEtherVPN/SoftEtherVPN#2016] [General]
Can I offer my help with the English language documentation, website, and app prompts? Not the coding, just the text.

**Comment by chipitsine** [comment]
we are open to contribution, let us figure out how it can work...

**Comment by connect2400** [comment]
great, please let me know how to proceed, who to contact and what is the best method... should I just look at the existing language I see publicly and email my improvements? or is there a more preferable way?

**Reply by chipitsine** [reply]
what's your native language ?

I've an idea, if you speak japanese, it would be nice to establish a bridge between people on https://www.vpnusers.com/
(however, I found google translate good enough)

**Comment by connect2400** [comment]
My native language is English, and I am an expert at grammar, etc. So although I can't really do translating, in using the app and site I saw many examples of poor English, and my idea was that just editing, rephrasing, and improving these existing things (which turn off some professional users) could help the project.

Sent with [Proton Mail](https://proton.me/) secure email.
